inc/deploycheck.class.php

```diff
@@ -226,14 +263,17 @@ static function displayList(PluginFusioninventoryDeployPackage $package, $datas,
       echo "<table class='tab_cadrehov package_item_list' id='table_check_$rand'>";
       $i = 0;
       foreach ($datas['jobs']['checks'] as $check) {
-         //specific case for filesystem size
-         if (is_numeric($check['value'])
-            && in_array($check['type'], ['freespaceGreater', 'fileSizeLower',
-                                         'fileSizeEquals', 'fileSizeGreater'])) {
-            if ($check['type'] == "freespaceGreater") {
+         switch ($check['type']) {
+            case 'freespaceGreater':
                $check['value'] = $check['value'] * 1024 * 1024;
-            }
-            $check['value'] = PluginFusioninventoryDeployFile::processFilesize($check['value']);
+               break;
```

Comment:
i'm not sure about this ```break;```.

Before, the ```1024 * 1024``` part was executed also with ``processFilesize`` part in case of freespaceGreater